buildSrc/subprojects/build/build.gradle.kts

```diff
@@ -1,7 +1,9 @@
 dependencies {
+    compile(project(":buildPlatform"))
```

Comment:
Can we use `implementation` instead of `compile`?

Reply:
Changed to `api` and `implementation` in a21645158db.